Products/CMFPlone/image_scales/interfaces.py

```diff
@@ -0,0 +1,23 @@
+from zope.interface import Interface
+
+
+class IImageScalesAdapter(Interface):
+    """
+    Return a list of image scales for the given context
+    """
+
+    def __init__(context, request):
+        """Adapts context and the request."""
+
+    def __call__():
+        """ """
+
+
+class IImageScalesFieldAdapter(Interface):
+    """ """
+
+    def __init__(field, context, request):
+        """Adapts field, context and request."""
+
+    def __call__():
+        """Returns JSON compatible python data."""
```

Comment:
The interfaces here need to go to plone.base before merge. @cekk put them here to simplify prototyping. It was never planned to keep them here.